gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c

```diff
@@ -816,54 +819,73 @@ _message_handler (void *thread_data)
 {
   nns_edge_thread_data_s *_tdata = (nns_edge_thread_data_s *) thread_data;
   nns_edge_handle_s *eh;
+  nns_edge_conn_s *conn;
+  int64_t client_id;
   nns_edge_cmd_buf_s cmd_buf;
+  char *val;
+  int ret;
   nns_edge_event_h event_h = NULL;
 
   eh = (nns_edge_handle_s *) _tdata->eh;
```

Comment:
How about null check about eh?

Reply:
Line 840 NNS_EDGE_MAGIC_IS_VALID (eh) will check null ptr, and next PR about edge-event will update condition.

Reply:
thank you.

Reply:
thank you.